cmd/dynamodb/main.go

```diff
@@ -28,7 +28,11 @@ func initDB() model.Router {
 		appStorage,
 		userStorage,
 	)
-	r := ihttp.NewRouter(nil, appStorage, userStorage, tokenStorage, tokenService)
+	r, err := ihttp.NewRouter(nil, appStorage, userStorage, tokenStorage, tokenService)
+
+	if err != nil {
+		log.Fata(err)
```

Comment:
Please compile your code before commiting. There is no function `Fata` in the `log` package.